frontend/packages/kubevirt-plugin/src/topology/TopologyVmPanel.tsx

```diff
@@ -45,10 +45,10 @@ export const ConnectedTopologyVmPanel: React.FC<TopologyVmPanelProps> = ({
   const vmData = vm.getData();
   const vmObj = vmData.resources.obj;
   const { namespace } = vmObj.metadata;
+  const actions = vmActions(vm.getData());
   const detailsComponent = () => <TopologyVmDetailsPanel vm={vm} />;
   const resourcesComponent = () => <TopologyVmResourcesPanel vmNode={vm} />;
 
-  const actions = vmActions(vm.getData());
   return (
     <div className="overview__sidebar-pane">
       <div className="overview__sidebar-pane-head resource-overview__heading">
```

Comment:
and pass the vm reference through tabProps. We can also unife prop names vm x vmNode